### DISCUSSION TITLE
PSA: upgrade to v5 killed tabs

### DISCUSSION THREAD

**Discussion by ubitux** [mbnuqw/sidebery#1239] [General]
Last upgrade (v5.0.0) caused all the tabs that were associated with a ~~workspace~~ container to disappear. The associated panels remained, but not these specific tabs.

**Comment by mbnuqw** [comment]
If by "workspace" you meant container, then it's probably because Firefox [can remove](https://bugzilla.mozilla.org/show_bug.cgi?id=1473799) containers (and close containered tabs) after disabling/removing of the last addon that uses those containers. But this doesn't happen (normally) when updating an addon (internally it's removing the old/installing the new), so probably a Firefox bug? At least from Sidebery perspective, nothing can be done here.
